rapids-4-spark-tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala

```diff
@@ -102,4 +104,78 @@ class ApplicationInfoSuite extends FunSuite with Logging {
     }
   }
 
+  test("test filename match") {
+    val matchFileName = "udf"
+    val appArgs = new ProfileArgs(Array(
+      "--match-event-logs",
+      matchFileName,
+      "src/test/resources/spark-events-qualification/udf_func_eventlog",
+      "src/test/resources/spark-events-qualification/udf_dataset_eventlog",
+      "src/test/resources/spark-events-qualification/dataset_eventlog"
+    ))
+
+    val result = ProfileMain.processAllPaths(appArgs.filterCriteria,
+      appArgs.matchEventLogs, appArgs.eventlog())
+    assert(result.length == 2)
+  }
+
+  val tempFile1 = File.createTempFile("tempOutputFile1", null)
```

Comment:
It would be nice if these were inside the tests to make sure they get cleaned up, if it fails somehow these temp files will be left around.  In the very least we should add deleteOnExit right away here.

Reply:
moved these to inside tests.